jars/cookies/content.py

```diff
@@ -68,10 +72,8 @@ def handle_bulk(file, form):
 
     # Each file will result in a new LocalResource.
     for name in z.namelist():
-    
         # Some archives have odd extra files, so we'll skip those.
-        if not name.startswith('._'):
-        
+        if not basename(name).startswith('._'):
```

Comment:
This is the fix. FIxing it by checking the basename of the file has ._ .
If not then only consider those files.